docs/common/object.ldoc

```diff
@@ -0,0 +1,31 @@
+
+--- Connect to a signal weakly. This allows the callback function to be garbage
+-- collected and automatically disconnects the signal when that happens.
```

Comment:
Add something like: Only use this function if you really, really, *really* know what you are doing.